submission/submission-security/src/main/kotlin/ebi/ac/uk/security/integration/SecurityModuleConfig.kt

```diff
@@ -30,9 +30,9 @@ class SecurityModuleConfig(
     fun groupService(): IGroupService = groupService
     fun securityFilter(): ISecurityFilter = securityFilter
 
-    val userRegister: Observable<UserRegister> = Events.userRegister
+    val userRegister: Observable<UserActivated> = Events.userRegister
```

Comment:
We should change the variable names as well just to avoid any reference to the previous preregister/register approach